openapi/components/schemas/KycDocument/IdentityMatches.yaml

```
@@ -80,6 +80,10 @@ properties:
     type: string
     nullable: true
     example: passport
+  hasMatchingFaceProof:
+    description: If identity document has a matching face proof.
```

nitpick: I think we should start boolean descriptions with 'specifies whether' 
```suggestion
    description: Specifies whether an identity document has a matching face proof.
```